frontend/src/components/ScheduledEventsTableCard/ScheduledEventsTableCard.tsx

```diff
@@ -1,14 +1,18 @@
 import React, { FunctionComponent, ReactNode, useEffect, useState } from 'react';
 import { Card } from 'react-bootstrap';
 import { Dimmer, Loader } from 'semantic-ui-react';
+import { fetchData, getScheduledEventsByPage } from '.';
 import { PaginationRow } from '../PaginationRow';
 import { ScheduledEventsTable } from '../ScheduledEventsTable';
-import { fetchData } from '.';
 
 const LIMIT = 5;
```

Comment:
You're creating this in too many places ... Have it in a single util file that you use throughout all related code ... as long as its tied to the same table. i.e. ScheduledEvents limit = 5, runhistory limit = 10